### ISSUE TITLE
npm uses vue-toastr.min.js by default

### ISSUE THREAD

**Issue by cloudybrain** [s4l1h/vue-toastr#38]
By making vue-toastr.min.js by default when using es6 imports. When comparing vue-toastr.min.js to vue-toastr.js, the code looks exactly the same

    import Toastr from 'vue-toastr'

I look at my source scripts and it uses vue-toastr.min.js, when switching to using vue-toastr.js it's the same code but with a mapping. This causes problems when I try to switch environments has I minify/uglify scripts. When the min file is already minified, it gets minified again and can't be used.

**Comment by s4l1h** [comment]
@cloudybrain can you try like this.

`import Toastr from 'vue-toastr/src/main'
`
Probably it will be working.

https://github.com/s4l1h/vue-toastr/blob/master/package.json#L5

We should change this line.

**Comment by cloudybrain** [comment]
Hi, apparently that got me on the right track, but I had to change my webpack so much because of the html and sass loaders, or some other things I might be missing in it. I just can't risk it at this point since I spent a lot of time trying to get it to work because of one library. I tried various hacks to see how to force it to stop erroring out, and I failed in the end. However, this library taught me how to incorporate vue plugins, so I was able to bring in my own toastr library and use it straight out of the box.
